### DISCUSSION TITLE
Embed lighthouse tool to page that should be tested.

### DISCUSSION THREAD

**Discussion by sdenisen** [GoogleChrome/lighthouse#14813] [General]
Hello, everybody!

Is it possible to embed the lighthouse tool to the page that should be tested? 
Is there some javascript that can be injected into the page and generate the lighthouse report from the browser console? 
is there are any suggestions on how it can be implemented? 

Thanks,

**Comment by adamraine** [comment]
This probably isn't feasible. Even if it was, injecting the entirety of Lighthouse into the page would affect the performance of that page so the results wouldn't be accurate.

**Comment by connorjclark** [comment]
Perhaps you just want the browser extension: https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=en
